Gemfile

```diff
@@ -32,4 +32,5 @@ group :test, :development do
   gem 'flamegraph',     '~> 0.0'
   gem 'rspec-prof',     '~> 0.0'
   gem 'byebug'
+  gem 'test-unit'
```

Comment:
Dat lost een rare rspec bug op onder nieuwe rubies, met de gem updates waar Arnold mee bezig is kan dit weer weg.